modules/mesh-layers/src/utils/matrix.js

```diff
@@ -153,3 +153,17 @@ export const MATRIX_ATTRIBUTES = {
     }
   }
 };
+
+export function shouldComposeModelMatrix(viewport, coordinateSystem) {
```

Comment:
Add comments here to document why the model matrix is only composed for these coordinate systems